nixos/modules/services/monitoring/dd-agent/dd-agent.nix

```diff
@@ -15,102 +15,48 @@ let
     collector_log_file: /var/log/datadog/collector.log
     forwarder_log_file: /var/log/datadog/forwarder.log
     dogstatsd_log_file: /var/log/datadog/dogstatsd.log
-    pup_log_file:       /var/log/datadog/pup.log
+    jmxfetch_log_file:  /var/log/datadog/jmxfetch.log
```

Comment:
Please provide a variable for the /var/log/datadog directory. Having a default is fine.